package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Pretty sure this file shouldn't exist?

Reply:
Yep looks like you did a npm install outside of the server folder.

Reply:
yep deleted